pallets/marketplace/src/tests.rs

```diff
@@ -293,12 +292,15 @@ fn offering_works() {
 			INSTANCE_ID_0,
 			metadata.clone()
 		));
+		/*
```

Comment:
there is nothing done with class 1 in the test so we can remove this assertion

Reply:
done